velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -268,27 +270,165 @@ TpchPlan TpchQueryBuilder::getQ6Plan() const {
   return context;
 }
 
-const std::vector<std::string> TpchQueryBuilder::kTableNames_ = {"lineitem"};
+TpchPlan TpchQueryBuilder::getQ18Plan() const {
+  static const std::vector<std::string> kTableNames = {
+      "lineitem", "orders", "customer"};
+  std::vector<std::string> lineitemSelectedColumns = {
+      "l_orderkey", "l_quantity"};
+  std::vector<std::string> ordersSelectedColumns = {
+      "o_orderkey", "o_custkey", "o_orderdate", "o_totalprice"};
+  std::vector<std::string> customerSelectedColumns = {"c_name", "c_custkey"};
+
+  // Get the corresponding selected column names in the file.
+  const auto lineitemAliasSelectedColumns =
```

Comment:
ditto: lineitemAliases, ordersAliases, customerAliases

Reply:
I agree, these can be shortened because of their localized context I think